goal_point: [[-9.65,-2.7,0],[0,0,0.6051864,0.7960838]]